### PR TITLE
Mongo Configmap to Secret Rewrite

### PR DESCRIPTION
**Pull request by bmheadley**
**Please provide enough information so that others can review your pull request:**

1. Explain the **details** for making this change. What existing problem does the pull request solve? If it resolves an existing issue, be sure to use a Github [keyword](https://help.github.com/en/articles/closing-issues-using-keywords) to automatically close it.
Fixes helm chart issue #486 about mongo configmap exposing too much information.
<br/><br/>

2. What **changes to custom.yaml** are required?
None
<br/><br/>

3. Have you documented any additional setup steps or configurations options?
No additional setup required. Configmap to secret rewrite. 
<br/><br/>

4. Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots / videos if the pull request changes UI.
Tested in devinabox. Exact helm install command used was 
helm install greymatter -f greymatter.yaml -f greymatter-secrets.yaml --name gm -f credentials.yaml
<br/><br/>

### CODE REVIEW

File: data/values.yaml [hunk 1]
```diff
@@ -42,11 +42,14 @@ data:
   client_jwt_endpoint_use_tls: 'true'
   base_path: /services/{{ $.Values.data.name | lower }}/{{ $.Values.data.version | trunc 3 }}
   image: docker.production.deciphernow.com/deciphernow/gm-data:{{ $.Values.data.version }}
-  imagePullSecret: docker.secret
   certs_mount_point: /certs
   imagePullPolicy: IfNotPresent
   use_tls: true
   master_key: ac8923[lkn43589vi23kl4rfgv0ws
+  user: 'gmdata-user'
```
Comment (chrisbsmith) [inline data/values.yaml]: Are these leftover from earlier testing?  They aren't used and should probably be removed.

**Comment by chrisbsmith** [comment]
All in all the change looks good.  However, we should port these changes into the `greymatter-secrets.yaml` file.  You need to add a `mongosecrets` section to `data` and `internal-data` sections.

File: data/values.yaml [hunk 2]
```diff
@@ -42,8 +42,8 @@ data:
   client_jwt_endpoint_use_tls: 'true'
   base_path: /services/{{ $.Values.data.name | lower }}/{{ $.Values.data.version | trunc 3 }}
   image: docker.production.deciphernow.com/deciphernow/gm-data:{{ $.Values.data.version }}
-  imagePullSecret: docker.secret
   certs_mount_point: /certs
+  imagePullSecret: docker.secret
```
Comment (chrisbsmith) [inline data/values.yaml]: I like what you did here, but I don't see it used anywhere.  Can you show me where it's being used?

Comment (bmheadley) [reply data/values.yaml]: I believe I took out docker.secret in some early testing. This was just replacing what was already there.

Comment (chrisbsmith) [reply data/values.yaml]: I see.  Do me a favor, to keep git happy, move this line back to line 45.  then github won't see it as a delta.

**Comment by moosecanswim** [comment]
Worked for me.

**Pull request merged**